src/encoded/static/components/__tests__/about-test.js

```diff
@@ -32,13 +32,68 @@ describe('Testing about.js', function() {
     });
 
 
-    // Check that has navBar with links
+    // Check that has functional navBar with links
     it('Has global navigation bar & links', function() {
 
         var navBanner = TestUtils.scryRenderedDOMComponentsWithClass(page, 'navbar navbar-main');
         var navBannerLinkWrapper = TestUtils.scryRenderedDOMComponentsWithClass(page, 'nav navbar-nav');
         expect(navBanner.length).toEqual(1);
         expect(navBannerLinkWrapper.length).toBeGreaterThan(0); // Doesn't matter if 1 or more links.
+
+        // Test mobile dropdown (full menu)
+        var menuToggleButton = navBanner[0].children[0].children[0]; // nav.navbar.navbar-main > div.navbar-header > a.navbar-toggle
+        var menu = navBanner[0].children[1]; // nav.navbar.navbar-main > div.navbar-collapse.collapse
+        expect(menuToggleButton.className.search('navbar-toggle')).toBeGreaterThan(-1);
+        expect(menuToggleButton.getAttribute('aria-expanded')).toEqual('false');
+        expect(menu.className.search('in')).toBe(-1);
```

Comment:
Hey Can I just ask about this.  Is it looking for a menu item with class name in to not be visible?  Is there a more english way to say that in the code?